nvidia-runtime.sysext/files/usr/local/bin/_nvidia-persistenced-helper

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+
+SYSEXT_MODE=0
+if systemd-sysext list 2>&1 | grep -q flatcar-nvidia-drivers; then
+  SYSEXT_MODE=1
+fi
+
+_pre() {
+  if [ "$SYSEXT_MODE" -eq 0 ]; then
+    rm /run/extensions/nvidia-driver && systemctl restart systemd-sysext || true
+  fi
+}
+
+_post() {
+  chcon -R -t container_file_t /dev/nvidia* || true
+
+  if [ "$SYSEXT_MODE" -eq 0 ]; then
+    /opt/bin/nvidia-persistenced || true
+    mkdir -p /run/extensions
+    # /opt/nvidia/current gets mounted as an overlay on root filesystem
+    mkdir -p /opt/nvidia/current/usr/bin && ln -sf /opt/bin/nvidia-smi /opt/nvidia/current/usr/bin/nvidia-smi
+    mkdir -p /opt/nvidia/current/usr/local/ && ln -sf /opt/nvidia /opt/nvidia/current/usr/local/nvidia
+    ln -sf /opt/nvidia/current /run/extensions/nvidia-driver
+    systemctl restart systemd-sysext
+  else
+    systemctl enable --now nvidia-persistenced
```

Comment:
i've been meaning to drop launching `nvidia-persistenced` - it causes issues for some usecases. Lets drop this line and leave this to users.